app/workers/rapid_ready_for_decision/rrd_processor.rb

```diff
@@ -63,23 +63,6 @@ def add_medical_stats(assessed_data)
       form526_submission.add_metadata(med_stats: med_stats_hash)
     end
 
-    def send_fast_track_engineer_email_for_testing(job_id, error_message, backtrace)
-      # TODO: This should be removed once we have basic metrics
-      # on this feature and the visibility is imporved.
-      body = <<~BODY
-        A claim errored in the #{Settings.vsp_environment} environment \
-        with Form 526 submission id: #{form526_submission.id} and Sidekiq job id: #{job_id}.<br/>
-        <br/>
-        The error was: #{error_message}. The backtrace was:\n #{backtrace.join(",<br/>\n ")}
-      BODY
-      ActionMailer::Base.mail(
-        from: ApplicationMailer.default[:from],
-        to: Settings.rrd.alerts.recipients,
-        subject: 'Rapid Ready for Decision (RRD) Job Errored',
-        body: body
-      ).deliver_now
-    end
-
```

Comment:
No longer used, in favor of `Form526RapidReadyForDecisionConcern.send_rrd_alert_email` instead.